src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts

```diff
@@ -230,11 +235,47 @@ if it is ++ or -- operater navigate to the previous or next tab
       ),
       enable: showPostIntakeTabs,
       state: {
+        ...(userIsMpo
+          ? {
+              next: {
+                condition: true,
+                action: +1,
+              },
+            }
+          : {}),
         prev: {
           condition: true,
           action: -1,
         },
       },
     },
+    {
+      id: 11,
+      isDivider: true, // divider
+      label: '',
+      link: '',
+      enable: userIsMpo && showPostIntakeTabs,
+    },
```

Comment:
I think you may need to put the divider inside the `userIsMpo` logic

Reply:
This will show up an empty line in the end, when the user is just a drafter

Reply:
Not so. I tested this exact case.